raiden-ts/src/raiden.ts

```diff
@@ -604,18 +605,18 @@ export class Raiden {
    * @returns Promise to list of token addresses
    */
   public async getTokenList(): Promise<Address[]> {
-    return this.deps.provider
-      .getLogs({
-        ...this.deps.registryContract.filters.TokenNetworkCreated(null, null),
-        fromBlock: this.deps.contractsInfo.TokenNetworkRegistry.block_number,
-        toBlock: 'latest',
-      })
-      .then((logs) =>
-        logs
-          .map((log) => this.deps.registryContract.interface.parseLog(log))
-          .filter((parsed) => !!parsed.args.token_address)
-          .map((parsed) => parsed.args.token_address as Address),
-      );
+    return await getLogsByChunk$(this.deps.provider, {
+      ...this.deps.registryContract.filters.TokenNetworkCreated(null, null),
+      fromBlock: this.deps.contractsInfo.TokenNetworkRegistry.block_number,
+      toBlock: await this.getBlockNumber(),
```

Comment:
Why do you change this from `latest` to blocknumber?

Reply:
`getLogsByChunk$` calculates the ranges on which to iterate based on `fromBlock` & `toBlock`, therefore it needs both to be numeric.